src/components/layout/chat/message.jsx

```diff
@@ -125,7 +125,7 @@ const Message = ({ message }) => {
       <div className={handleMessageType(message)}>
         {handleBadges(message)}
         <span className="chat-user-name">{`${handleMessageSender(message)}${
-          message.type === types.self ? "" : ":"
+          message.type === roles.self ? "" : ":"
```

Comment:
keep message.type the same. 
Sorry didn't think about this one.

Reply:
Fixed. Sorry I guess I misunderstood how this files `types` were handled.